### ISSUE TITLE
Setting Database Results

### ISSUE THREAD

**Issue by NogalskiM** [jongpie/NebulaLogger#115]
When I add logs that take Database Results as a parameter and User executing the code doesn't have logging enabled (for example, in a test class that don't have LoggerSettings__c initialized), code is failing with exception.

```
System.NullPointerException: Attempt to de-reference a null object
Class.LogEntryEventBuilder.setDatabaseResult: line 187, column 1
Class.Logger.warn: line 969, column 1
```

The reason for this it that LogEntryEventBuilder.setDatabaseResult methods are not checking, if logging is enabled.
The missing code is:

```
if (this.shouldSave == false) {
    return this;
}
```

Without this code, method is trying to reference this.logEntryEvent.DatabaseResultJson__c, which is not initialized - LogEntryEventBuilder constructor also have condition, that it doesn't initialise logEntryEvent when shouldSave variable is set to False

**Comment by jongpie** [comment]
Hey @NogalskiM - thanks for reporting this issue! I have an open PR, #114, that adds support for lists of database results - I've also included a bugfix for this issue within that PR. I'm hoping to have it merged sometime this week.

Let me know if you find any other issues!

**Comment by jongpie** [comment]
@NogalskiM I've merged PR #114 to `master`, which includes a bugfix for this. If you come across any other issues, let me know!
